### DISCUSSION TITLE
Org-mode and table.el tables (maybe bug?)

### DISCUSSION THREAD

**Discussion by buhtz** [pygments/pygments#2688] [General]
Emacs org-mode do support org-tables and table.el tables ([ref](https://orgmode.org/worg/org-syntax.html#Tables)).
In the following screenshot you see one "table.el" style tabel in the beginning and a "org-table" as second using the Org-mode lexer.

![image](https://github.com/pygments/pygments/assets/11861496/10a25772-b02a-4a2c-bf20-cef2c536d3eb)

I am not sure but it seems that the org-mode lexer is not able to handle the table.el style table correct. Am I right? Can we treat this as a feature request? To my understanding the org-mode lexer should support table.el tables because they are part of the org-mode specification (see ref link above).

Or is there a better lexer for table.el tables?

Asking the other way around? Is there a void-lexer? A lexer that does nothing, no color.

**Comment by birkenfeld** [comment]
Sure, it looks like a bug/feature request (such thing are hard to properly distinguish in this context). The best option to get it implemented is probably to ping the author of the org mode lexer, @goshatch 

For the second question, yes, the Text lexer just emits a Text token for everything.

**Comment by goshatch** [comment]
Hello! I implemented the original org mode lexer, and yes, it does not have support for tables at the moment. As I don't have the bandwidth to work on this in the foreseeable future, would you be able to make a PR to add this?

**Reply by buhtz** [reply]
I have no idea how to create a lexer. Sorry.
